dev-resources/private/test/src/cljs/foo/load_require.cljs

```diff
@@ -0,0 +1,6 @@
+(ns foo.load-require)
+
+;;; a repl special
+(require 'foo.bar.baz)
+
```

Comment:
but it's not true, because `(load-file "foo/load_require")` will fail because there is thre `requrie` repl special, this is the only purpose of this file/namespace